docs/sources/release-notes/v2-3.md

```diff
@@ -42,7 +42,7 @@ Lastly several useful additions to the LogQL query language have been included:
 
 ## Upgrade considerations
 
-The path from 2.2.1 to 2.3.0 should be smooth, as always, read the [Upgrade Guide](https://github.com/grafana/loki/blob/master/docs/sources/upgrading/_index.md#230) for important upgrade guidance.
+The path from 2.2.1 to 2.3.0 should be smooth, as always, read the [Upgrade Guide](https://github.com/grafana/loki/blob/master/docs/sources/setup/upgrade/_index.md#230) for important upgrade guidance.
```

Comment:
I think this should be a relref to the page in technical documentation and not a link to the GitHub source file.